tests/test_datacard.py

```diff
@@ -18,14 +18,48 @@
 BUILD = audeer.path('..', 'build', 'html')
 
 
-def test_datacard_example(db, cache):
+@pytest.mark.parametrize(
+    'db',
+    [
+        'minimal_db',
+        'medium_db',
+    ],
+)
+def test_datacard(db, cache, request):
+    """Test datacard creation from jinja2 templates."""
+    db = request.getfixturevalue(db)
```

Comment:
This will return the name of the database as a string, right? 
For me, I understand `db` as something like `self` or class, i.e. a kind of strong convention for naming things, in this case an instance of what is returned when you do `audb.load` (`audformat.Database` or so). I would therefore prefer to use sth. like `dbname`.

Reply:
No, it will indeed return an `audformat.Database` object. A few lines later I use `db.name`.
So everything should be in line with your expectations.